<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item row="0" column="2" alignment="Qt::AlignmentFlag::AlignRight">
    <widget class="QToolButton" name="toolButton_close">
     <property name="styleSheet">
      <string notr="true">QToolButton {
    border: none;
    background: transparent;
	image: url(:/images/close_b.png);
    color: rgb(78, 78, 78);
    font:  16pt 'Microsoft New Tai Lue';
}

QToolButton:hover {
    border: none;
    background: transparent;
	image: url(:/images/close_w.png);
    color: white;
    font:  16pt 'Microsoft New Tai Lue';
    background: red;
    border-radius: 3px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>84</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="lab_avatar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid white;
border-radius: 50px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pu_login">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>298</width>
       <height>37</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>298</width>
       <height>37</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    font: 12pt 'Microsoft YaHei UI';
    color: white;
    border-radius: 15px;
    background:
        linear-gradient(to bottom right, rgba(255,255,255,0.2), transparent 50%),
        qlineargradient(x1: 1, y1: 0, x2: 1, y2: 1, stop: 0 #c0d0ff, stop: 1 #a0b0e0);
}</string>
     </property>
     <property name="text">
      <string>安全登录</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="LineEdit" name="linee_usernum">
       <property name="minimumSize">
        <size>
         <width>289</width>
         <height>37</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
border: 1px solid rgba(0, 0, 0, 0.3);
border-radius: 10px;
color:grey;
</string>
       </property>
       <property name="text">
        <string>请输入账号</string>
       </property>
       <property name="maxLength">
        <number>12</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineEdit" name="linee_password">
       <property name="minimumSize">
        <size>
         <width>289</width>
         <height>37</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Microsoft YaHei UI&quot;;
border: 1px solid rgba(0, 0, 0, 0.3);
border-radius: 10px;
color:grey;
</string>
       </property>
       <property name="text">
        <string>请输入密码</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>40</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignmentFlag::AlignRight">
        <widget class="QCheckBox" name="ch_rpwd">
         <property name="styleSheet">
          <string notr="true"> color: gray;</string>
         </property>
         <property name="text">
          <string>记住密码</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignLeft">
        <widget class="QPushButton" name="pu_fpwd">
         <property name="styleSheet">
          <string notr="true"> QPushButton {
       font: 10pt &quot;Microsoft YaHei UI&quot;;
       border: none;
       text-decoration: none; 
      color: gray;
   }

   QPushButton:hover {
       text-decoration: underline; 
       color: gray;
   }</string>
         </property>
         <property name="text">
          <string>忘记密码</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item alignment="Qt::AlignmentFlag::AlignTop">
      <widget class="QLabel" name="lab_logo">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_logo_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>赛搏</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>90</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pu_register">
       <property name="styleSheet">
        <string notr="true">QPushButton {
       font: 10pt &quot;Microsoft YaHei UI&quot;;
       border: none;
       text-decoration: none; 
       color: gray;
   }

   QPushButton:hover {
       text-decoration: underline; 
       color: gray;
   }</string>
       </property>
       <property name="text">
        <string>注册账号</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>rewritten_widgets/lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
